pkg/logs/input/file/position.go

```diff
@@ -12,25 +12,82 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/auditor"
 )
 
+// TailingMode type
+type TailingMode uint8
+
+// Tailing Modes
+const (
+	ForceBeginning = iota
+	ForceEnd
+	Beginning
+	End
+)
+
+// Tailing mode string representation
+const (
+	ForceBeginningStr = "forceBeginning"
+	ForceEndStr       = "forceEnd"
+	BeginningStr      = "beginning"
+	EndStr            = "end"
+)
+
+var tailingModeToStringRepresentations = map[TailingMode]string{
+	ForceBeginning: ForceBeginningStr,
+	ForceEnd:       ForceEndStr,
+	Beginning:      BeginningStr,
+	End:            EndStr,
+}
+
+// TailingModeFromString parses a string and returns a corresponding tailing mode, default to End if not found
+func TailingModeFromString(mode string) (TailingMode, bool) {
+	for m, mStr := range tailingModeToStringRepresentations {
+		if mStr == mode {
+			return m, true
+		}
+	}
+	return End, false
+}
+
+// TailingModeToString returns seelog string representation for a specified tailing mode. Returns "" for invalid tailing mode.
+func (mode TailingMode) String() string {
+	modeStr, ok := tailingModeToStringRepresentations[mode]
+	if ok {
+		return modeStr
+	}
+	return ""
+}
+
 // Position returns the position from where logs should be collected.
-func Position(registry auditor.Registry, identifier string, tailFromBeginning bool) (int64, int, error) {
+func Position(registry auditor.Registry, identifier string, m TailingMode) (int64, int, error) {
```

Comment:
nitsL if we want to be consistent with the rest of the code
```suggestion
func Position(registry auditor.Registry, identifier string, mode TailingMode) (int64, int, error) {
```

Reply:
It makes sense and slightly improves the readability.